File: Todos/C/BEE 3209.c
/*
  autor: Vinist021;
  data: 19/02/2025;
  nome: Tomadas Elétricas;
*/

#include<stdio.h>

int main()
{
    int N;

    scanf("%d", &N);

    for(int i = 0; i < N; i++)
    {
        int K, qtd, total = 0;

        scanf("%d", &K);

        for(int j = 0; j < K; j++)
        {
            scanf("%d", &qtd);
            total += qtd;
        }
        total = total - (K-1);
        printf("%d\n", total);
    }

    return 0;
}

File: Todos/Python/BEE 3209.py
#   autor: Vinist021;
#   data: 29/02/2025;
#   nome: Tomadas Elétricas;

N = int(input())

for _ in range(N):
    l = list(map(int, input().split()))
    K = l[0]
    l.pop(0)

    total = 0
    for i in l:
        total += i

    total = total - (K-1)
    print(total)